packages/venia-ui/lib/components/ProductImageCarousel/carousel.js

```diff
@@ -35,7 +35,8 @@ const ProductImageCarousel = props => {
     const { images } = props;
 
     const talonProps = useProductImageCarousel({
-        images
+        images,
+        containerWidth: DEFAULT_IMAGE_WIDTH
```

Comment:
I have concerns about `containerWidth` - what if there are other elements in the container?

Reply:
In this particular example we're setting `containerWidth` to the image's "default" width. I suppose I was thinking of it like

```html
<div id="container">
  <img />
</div>
```

If there are other things in `#container`, then `containerWidth` is not going to match `DEFAULT_IMAGE_WIDTH`.

Am I thinking about `containerWidth` incorrectly?

Reply:
My bad, I meant the image container width, the width of the `<img />` tag. I will be using this width to calculate the probable `srcset` URL that might be downloaded in the future.

Reply:
Oh dang, ok - can we call it `imageWidth` or something then?

Reply:
Agree with @supernova-at here. Consider `imageWidth` or `defaultImageWidth`.